mocha.el

```diff
@@ -70,7 +70,7 @@ From http://benhollis.net/blog/2015/12/20/nodejs-stack-traces-in-emacs-compilati
 
 (defun mocha-find-project-root ()
   "Find the root of the project."
-  (let ((root-files '(".git" ".hg" ".svn" "package.json")) (dir nil) (i 0))
+  (let ((root-files '("package.json" ".git" ".hg" ".svn")) (dir nil) (i 0))
```

Comment:
I think checking for `package.json` or `node_modules` would be the best. My point (which I didn't express very clear) is that I don't think source control management directories needs to be used.